hw06-ATM/src/main/java/ru/Atm.java

```diff
@@ -0,0 +1,11 @@
+package ru;
+
+import java.util.HashSet;
+
+public interface Atm {
+    public HashSet<BanknoteCell> putBanknotes(Integer sum);
```

Comment:
```suggestion
    public Set<BanknoteCell> putBanknotes(Integer sum);
```

Программируйте на уровне интерфейсов.

Есть еще подобные места - о них уже не пишу.

Reply:
спасибо. буду стараться обращать внимание на эту проблему. часто  у меня это проскакивает.